<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="_13">
   <property name="spacing">
    <number>20</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tabRegular">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>&amp;Regular</string>
      </attribute>
      <layout class="QVBoxLayout" name="_8">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBasics">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Basics</string>
         </property>
         <layout class="QVBoxLayout" name="_10">
          <item>
           <widget class="QCheckBox" name="cbPreventSleep">
            <property name="text">
             <string>Prevent this computer from going to sleep</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbLanguageSync">
            <property name="text">
             <string>Use server's keyboard language on this computer (client mode)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbScrollDirection">
            <property name="text">
             <string>Invert scroll direction on this computer (client mode)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupApp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>App</string>
         </property>
         <layout class="QVBoxLayout" name="_9">
          <item>
           <widget class="QCheckBox" name="cbAutoUpdate">
            <property name="text">
             <string>Check for updates on startup </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbCloseToTray">
            <property name="text">
             <string>Leave app running in notification area when the window is closed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbAutoHide">
            <property name="text">
             <string>Hide the window when the app starts</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="lblTrayIconStyle">
              <property name="text">
               <string>Tray icon style</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbIconColorful">
              <property name="text">
               <string>Colorful</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbIconMono">
              <property name="text">
               <string>Monocolor</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupSecurity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Enable TLS Encryption</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="_11">
          <item>
           <widget class="QWidget" name="widgetTlsCert" native="true">
            <layout class="QHBoxLayout" name="_12">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lblTlsCert">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Certificate</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="lineTlsCertPath">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnTlsCertPath">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lblTlsKeyLength">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Key length</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboTlsKeyLength">
              <item>
               <property name="text">
                <string>1024</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2048</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4096</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnTlsRegenCert">
              <property name="text">
               <string>Regenerate certificate</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="cbRequireClientCert">
            <property name="text">
             <string>Require client certificates</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>&amp;Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="_2">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupNetworking">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Networking</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblPort">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbPort">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>24800</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblNetworkIp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Network IP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineInterface">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupLogs">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Logs</string>
         </property>
         <layout class="QGridLayout" name="_3">
          <property name="topMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <property name="verticalSpacing">
           <number>7</number>
          </property>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbLogToFile">
            <property name="text">
             <string>Log to file</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="comboLogLevel">
            <item>
             <property name="text">
              <string>Info</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Debug</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Debug1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Debug2</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="lblLogLevel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Level</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="QWidget" name="widgetLogFilename" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="_4">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="labelLogPath">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Log path</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="lineLogFilename">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnBrowseLog">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupService">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Service</string>
         </property>
         <layout class="QGridLayout" name="_6">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbServiceEnabled">
            <property name="toolTip">
             <string>Whether to launch the server or client process through the Windows background
                service or direct from the GUI.</string>
            </property>
            <property name="whatsThis">
             <string>
              &lt;p&gt;The background service is only available on Windows.&lt;/p&gt;
              &lt;p&gt;The Windows background service is used to:&lt;/p&gt;
              &lt;ul&gt;
              &lt;li&gt;Start the server or client automatically when the computer starts.&lt;/li&gt;
              &lt;li&gt;Run the server or client in an elevated mode (e.g. on login screen, UAC dialogs, etc).&lt;/li&gt;
              &lt;/ul&gt;
              </string>
            </property>
            <property name="text">
             <string>Use background service (daemon)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="widgetElevate" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="_7">
             <item>
              <widget class="QLabel" name="lblElevate">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Launch with elevated privileges</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboElevate">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Specify when the Windows background service should run the server or client
                  process at an elevated privilege level.</string>
               </property>
               <property name="whatsThis">
                <string>
                &lt;p&gt;You may want to alter whether or not the privilege level of the server or client 
                  process is automatically changed depending on your use case. In some cases it can help
                  to diagnose or solve some problems related to elevated processes in Windows.&lt;/p&gt;
                &lt;ul&gt;
                  &lt;li&gt;Automatic: Elevate when the window session changes to secure mode&lt;/li&gt;
                  &lt;li&gt;Always elevate: Always run in elevated mode (could be unsafe)&lt;/li&gt;
                  &lt;li&gt;Never elevate: Turn off compatibility with login screen and UAC&lt;/li&gt;
                &lt;/ul&gt;
                </string>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <item>
                <property name="text">
                 <string>Automatic (as needed)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Always elevate</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Never elevate</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupScope">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Use settings profile from</string>
         </property>
         <layout class="QHBoxLayout" name="_5">
          <item>
           <widget class="QRadioButton" name="rbScopeUser">
            <property name="text">
             <string>Current user</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbScopeSystem">
            <property name="text">
             <string>All users</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cbPreventSleep</tabstop>
  <tabstop>cbLanguageSync</tabstop>
  <tabstop>cbScrollDirection</tabstop>
  <tabstop>cbAutoUpdate</tabstop>
  <tabstop>cbCloseToTray</tabstop>
  <tabstop>cbAutoHide</tabstop>
  <tabstop>rbIconColorful</tabstop>
  <tabstop>rbIconMono</tabstop>
  <tabstop>groupSecurity</tabstop>
  <tabstop>lineTlsCertPath</tabstop>
  <tabstop>btnTlsCertPath</tabstop>
  <tabstop>comboTlsKeyLength</tabstop>
  <tabstop>btnTlsRegenCert</tabstop>
  <tabstop>cbRequireClientCert</tabstop>
  <tabstop>sbPort</tabstop>
  <tabstop>lineInterface</tabstop>
  <tabstop>cbLogToFile</tabstop>
  <tabstop>comboLogLevel</tabstop>
  <tabstop>lineLogFilename</tabstop>
  <tabstop>btnBrowseLog</tabstop>
  <tabstop>cbServiceEnabled</tabstop>
  <tabstop>comboElevate</tabstop>
  <tabstop>rbScopeUser</tabstop>
  <tabstop>rbScopeSystem</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
